test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -661,7 +661,7 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 					"kernel.sched_rt_runtime_us":    "950000",
 					"vm.stat_interval":              "10",
 				}
```

Comment:
Just checking: is this an exhaustive list? Because intel_pstate=disable should be present for high power consumption case. As should idle=poll iirc.

Reply:
How did this testcase pass before you made this change? Prior to your change, it seems like "intel_pstate=disable" would not have been present in this case, but it looks like this code is checking for it to be there? Or am I misunderstanding how this testcase works?

Reply:
It was working because before this change template was:
`{{if .HighPowerConsumption}}
cmdline_pstate=+intel_pstate=disable
{{end}}`

Now is:
`{{if .PerPodPowerManagement}}
cmdline_pstate=+intel_pstate=passive
{{else if .RealTimeHint}}
cmdline_pstate=+intel_pstate=disable
{{end}}
`

Reply:
Right - thanks Mario.

Reply:
Oh, right. Then I just wonder.. do we even allow high power consumption = true with realtime = false? This test might be code correct, but semantically broken if we do not.

Reply:
Well, it is allowed to do that, maybe show a warning?

Reply:
Nope, fine for now, we can rethink that separately.